src/isa.rs

```diff
@@ -90,30 +89,49 @@ pub struct DropKeep {
 	pub keep: Keep,
 }
 
-#[derive(Debug, Clone, PartialEq, Eq)]
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
 pub struct Target {
 	pub dst_pc: u32,
 	pub drop_keep: DropKeep,
 }
 
 /// A relocation entry that specifies.
-#[derive(Debug)]
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
 pub enum Reloc {
 	/// Patch the destination of the branch instruction (br, br_eqz, br_nez)
 	/// at the specified pc.
-	Br {
-		pc: u32,
-	},
+	Br { pc: u32 },
 	/// Patch the specified destination index inside of br_table instruction at
 	/// the specified pc.
-	BrTable {
-		pc: u32,
-		idx: usize,
-	},
+	BrTable { pc: u32, idx: usize },
+}
+
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub struct Targets<'a> {
```

Comment:
Since this is only for `br_table` and will not be used for other cases in the foreseen future, maybe it would be better  to rename this to `BrTargets`?